tests/use_cases/test_get_company_transactions.py

```diff
@@ -222,21 +225,27 @@ def test_that_correct_info_for_company_is_generated_in_correct_order_after_sever
     transaction_generator.create_transaction(
         sending_account=company1.means_account,
         receiving_account=company2.product_account,
+        date=transaction_generator.datetime_service.now() - timedelta(hours=5),
```

Comment:
Here again, you should inject `datetime_service: FakeDatetimeService` and use `datetime_service`.